roles/adf_app/molecule/default/host_vars/adf_app-instance.yml

```diff
@@ -1 +1,14 @@
 ansible_user: ansible
+acc_version: 9.3.0
+acc_artifact_id: alfresco-control-center
+acc_repository: >-
+  https://artifacts.alfresco.com/nexus/content/groups/public/org/alfresco
+acc_url: >-
+  {{ acc_repository }}/{{ acc_artifact_id }}/{{ acc_version }}/{{ acc_artifact_id }}-{{ acc_version }}.zip
+acc_checksum: sha1:{{ acc_url }}.sha1
```

Comment:
don't make hostvars and then include them in converge playbook, just hardcode them in the converge playbook